shoebot/grammar/nodebox.py

```diff
@@ -136,9 +135,11 @@ def image(
 
     def imagesize(self, path):
         """
-        :param: path    Path to image file.
-        :return: image size as tuple (width, height)
+        :param path: Path to image file.
+        :return: image size as a (width, height) tuple
         """
+        from PIL import Image as PILImage
+
         img = PILImage.open(path)
         return img.size
```

Comment:
It could be good to use a context manager at the end of this:

```python
with PILImage.open(path) as img:
    return img.size
```

Reply:
sure, why not, i'll put that in